### PR TITLE
chore: remove files for review

### PR DESCRIPTION
**Pull request by ro-ka**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,89 +0,0 @@
...
-
-All classes extending the `ThrottledTransform` class must implement the method `_throttledTransform`.  
-They may implement `_throttledFlush`, although this is not required.
-
-### API
-
-**ThrottledTransform.create(transform, flush = function(done) { done(); }, defaultOptions = {})**
-
-* `transform` `<Function>` The \_transform function of the stream. [See below](#api-for-extending-throttledtransform) for more details
-* `flush` `<Function>` The \_flush function of the stream. [See below](#api-for-extending-throttledtransform) for more details
```
Comment (ro-ka) [inline README.md:53]: 👍

File: README.md [hunk 2]
```diff
@@ -1,89 +0,0 @@
...
-* `defaultOptions` `<Object>` Default options for the class constructor
```
Comment (ro-ka) [inline README.md:54]: What are the options?

Comment (ro-ka) [inline README.md:54]: It it those explained in the next section?

File: src/throttled-transform.js [hunk 3]
```diff
@@ -1,188 +0,0 @@
-/* eslint-disable no-underscore-dangle */
-// TODO: use NPM package
-import ParallelTransform from '../../node-parallel-transform-stream/dist/parallel-transform'; // eslint-disable-line max-len
```
Comment (ro-ka) [inline src/throttled-transform.js:3]: Just to have it in the backlog: Use npm package.

File: src/throttled-transform.js [hunk 4]
```diff
@@ -1,188 +0,0 @@
...
-
-const _qps = new WeakMap(),
-  _bucketRunning = new WeakMap(),
-  _queries = new WeakMap(),
-  _queue = new WeakMap();
-
-export default class ThrottledTransform extends ParallelTransform {
-  /**
-   * ThrottledTransform instance
-   * All child class must implement the `_throttledTransform` function.
```
Comment (ro-ka) [inline src/throttled-transform.js:13]: Typo: _child classes_

File: src/throttled-transform.js [hunk 5]
```diff
@@ -1,188 +0,0 @@
...
-   * @returns {Class} A ParallelTransform class
-   **/
-  static create(transform, flush = done => done(), defaultOptions = {}) {
-    class Transform extends ThrottledTransform {
-      constructor(options) {
-        super(Object.assign({}, defaultOptions, options));
-      }
-
-      _throttledTransform = transform;
-      _throttledFlush = flush;
```
Comment (ro-ka) [inline src/throttled-transform.js]: No `this._throttledFlush`?

File: README.md (hunk 2 above)
Comment (Caerostris) [inline README.md:54]: Is it more clear like this? b41cacc

File: src/throttled-transform.js [hunk 6]
```diff
@@ -1,188 +0,0 @@
...
-    return Transform;
-  }
-
-  /**
-   * Executed every time data was pushed into the stream
-   * @param {?}        data Data to be transformed
-   * @param {string}   encoding Encoding, if `chunk` is a string
-   * @param {Function} done Callback which must be executed
-   *                        when transformations have finished
-   **/
```
Comment (ro-ka) [inline src/throttled-transform.js:60]: Wrong params in the DocBlock. There’s only `query`.

Comment (ro-ka) [inline src/throttled-transform.js:60]: Or is `query` the sum of the params and better defined as `params`?

File: src/throttled-transform.js [hunk 7]
```diff
@@ -1,188 +0,0 @@
...
-  /**
-   * Run this query, either unthrottled
-   * (if _skipThrottle is implemented and returned true), or throttled.
-   * @param {?}        data Data to be transformed
-   * @param {string}   encoding Encoding, if `chunk` is a string
-   * @param {Function} done Callback which must be executed
-   *                        when transformations have finished
-   * @returns {?} Undefined value to stop execution
-   **/
-  _processQuery(...query) {
```
Comment (ro-ka) [inline src/throttled-transform.js:74]: Same DocBlock param thing as above.

File: src/throttled-transform.js (hunk 6 above)
Comment (Caerostris) [inline src/throttled-transform.js:60]: Hmm, `query` is using the rest-operator, so the function technically accepts 3 arguments, not just "query".

File: src/throttled-transform.js [hunk 8]
```diff
@@ -1,188 +0,0 @@
...
-   * Check if this query can be processed without an asynchronous operation.
-   * Pushes the result if this succeeded.
-   * @param {?}        data Data to be transformed
-   * @param {string}   encoding Encoding, if `chunk` is a string
-   * @param {Function} done Callback which must be executed
-   *                        when transformations have finished
-   * @returns {boolean} True if the query could be completed
-   *                    wth a synchronous request
-   **/
-  _unthrottled(data, encoding, done) {
```
Comment (ro-ka) [inline src/throttled-transform.js:88]: To be consistent name this `_runUnthrottled`.

File: src/throttled-transform.js [hunk 9]
```diff
@@ -1,188 +0,0 @@
...
-   * @param {?}        data Data to be transformed
-   * @param {string}   encoding Encoding, if `chunk` is a string
-   * @param {Function} done Callback which must be executed
-   *                        when transformations have finished
-   * @returns {?} Undefined value to stop execution
-   **/
-  _runThrottled(...query) {
-    if (!_bucketRunning.get(this)) {
-      this._startBucket();
-    } else if (_queries.get(this) >= _qps.get(this)) {
```
Comment (ro-ka) [inline src/throttled-transform.js:110]: I think `_queries` should better be name something like `_queriesRunning`.

File: test/throttled-transform.js [hunk 10]
```diff
@@ -1,111 +0,0 @@
...
-  transformInstance.end();
-});
-
-test('should throw an error when not implementing _throttledTransform', t => {
-  class BrokenThrottledTransform extends ThrottledTransform {}
-  const transformInstance = new BrokenThrottledTransform();
-
-  t.throws(() => {
-    transformInstance.write('some data');
-  }, 'Not implemented');
```
Comment (ro-ka) [inline test/throttled-transform.js:46]: Maybe print out the name of the not implemented function, too. When it quits that info is nice to have.

Comment (ro-ka) [inline test/throttled-transform.js:46]: Applies to the parallel stream, too.

File: test/throttled-transform.js [hunk 11]
```diff
@@ -1,111 +0,0 @@
...
-});
-
-test('should allow instantiation via function call', t => {
-  const TStream = ThrottledTransform.create(1, (data, encoding, done) => {
```
Comment (ro-ka) [inline test/throttled-transform.js:50]: Rename to `ThrottledStream`.

**Comment by ro-ka** [comment]
🏁

File: src/throttled-transform.js (hunk 6 above)
Comment (ro-ka) [inline src/throttled-transform.js:60]: Yes, got it. Was irritated at first.

**Pull request closed without merging**